gunicorn.conf.py

```diff
@@ -1,38 +0,0 @@
-# -*- coding: utf-8 -*-
-import os
-import socket
-
-if 'GUNICORN_TIMEOUT' in os.environ:
-    timeout = int(os.environ['GUNICORN_TIMEOUT'])
-
-# Smart detect heroku stack and assume a trusted proxy.
-# This is a convenience that should hopefully not be too surprising.
-if 'heroku' in os.environ.get('LD_LIBRARY_PATH', ''):
-    forwarded_allow_ips = '*'
-
-if 'H_GUNICORN_CERTFILE' in os.environ:
-    certfile = os.environ['H_GUNICORN_CERTFILE']
-
-if 'H_GUNICORN_KEYFILE' in os.environ:
-    keyfile = os.environ['H_GUNICORN_KEYFILE']
```

Comment:
These only got copied into the new dev config files, not the prod ones. These settings are only meant to be used by `make devssl`